examples/cart/index.js

```diff
@@ -52,7 +52,7 @@ $(function() {
 
     attachCheckoutButtonListeners();
 
-    closeCart();
+    bindEventListeners();
```

Comment:
This should probably include the other listeners as well. Right now there's a bunch of specific functions being called, then one generic that is named like it would include all of them, but doesn't 😕